falcon/request.py

```diff
@@ -514,6 +522,58 @@ def cookies(self):
 
         return self._cookies.copy()
 
+    def param(self):
```

Comment:
@kgriffs my aim is to get this done this upcoming weekend, if possible

--
Rami Chowdhury
"Be the change you wish to see in the world." -- Mahatma Gandhi
+1-408-372-6496 / +44-7581-430517 / +88-1778-986280
On Oct 2, 2015 3:15 PM, "Kurt Griffiths" <notifications@github.com> wrote:

> In falcon/request.py
> <https://github.com/falconry/falcon/pull/493#discussion_r41057250>:
>
> > @@ -514,6 +522,58 @@ def cookies(self):
> >
> >          return self._cookies.copy()
> >
> > +    def param(self):
>
> OK, let's try it and see how it looks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/falconry/falcon/pull/493/files#r41057250>.
>